eth_tester/main.py

```diff
@@ -606,3 +606,59 @@ def get_all_filter_logs(self, filter_id):
 
         for item in filter.get_all():
             yield normalize_fn(item)
+
+    @to_tuple
+    def get_logs_directly(self, from_block=None, to_block=None, address=None, topics=None):
```

Comment:
Thoughts on just naming this `get_logs`?

Reply:
Agree.
Thank you for this great idea :D